.scripts/assets-manager/copy.mjs

```diff
@@ -47,14 +47,14 @@ glob(config.source).then((files) => {
     let baseFolder;
 
     if (config.source.indexOf("**") > 0) {
-        baseFolder = config.source.substring(0, config.source.indexOf("**"));
+        baseFolder = path.normalize(config.source.substring(0, config.source.indexOf("**")));
```

Comment:
Don't you need to normalize the returned path before passing it to path. path.normalize()?